### PR TITLE
chore(Expandable Section): convert examples to TypeScript

### LINKED ISSUE #7455: Expandable section: convert examples to TypeScript

### PR DESCRIPTION
**Pull request by jenny-s51**
**What**: Closes #7455

### CODE REVIEW

File: packages/react-core/src/components/ExpandableSection/examples/ExpandableSectionCustom.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ExpandableSection, Badge } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+export const ExpandableSectionCustom: React.FunctionComponent = () => {
```
Comment (thatblindgeye) [inline packages/react-core/src/components/ExpandableSection/examples/ExpandableSectionCustom.tsx:5]: Nitpick: the file name + exported name could instead be `ExpandableSectionCustomToggle` to provide more context of the example.

**Review by thatblindgeye — APPROVED** [review]
Looks good! Had a minor nitpick below, but it's more personal preference I think and not something I'd block over.

**Review by jenny-s51 — COMMENTED** [review]
Thank you @thatblindgeye! Renamed the custom toggle demo per your feedback

**Review by thatblindgeye — APPROVED** [review]
🎉 Approving again for the latest update

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**